### ISSUE TITLE
📍 Mazemap Marker Prop

### ISSUE THREAD

**Issue by xleonx0x** [lachlanshoesmith/mazemap-react#3]
When I try to use the Marker prop, at first there was a type error for the Markertype enum, I tried to circumvent this by putting the raw enum value (1, 2) which got rid of the type error but there was no marker 😢

**Comment by lachlanshoesmith** [comment]
> at first there was a type error for the Markertype

did you try `import`ing the `Marker` type as well as the `MazeMap` component itself?

and which other props were you using?

**Comment by xleonx0x** [comment]
I'll try this and update 👍

**Comment by xleonx0x** [comment]
it got rid of the typeerror but it still doesn't work 😢
```
<ReactPannellum
              id="1"
              sceneId="scene1"
              style={style}
              imageSource={levels[0]}
              config={config}
              onPanoramaLoaded={() => {
                restartTimer();
                setPanoramaLoaded(true);
              }}
              marker={{
                type: Marker.POIMarker,
                colour: '#ff0000',
                innerColour: '#0000ff',
                size: 32
              }}
            />
```

**Comment by lachlanshoesmith** [comment]
you seem to be using these props on the wrong component 😅

**Comment by xleonx0x** [comment]
oh whoops

**Comment by xleonx0x** [comment]
what no sleep does to u

**Comment by xleonx0x** [comment]
still same error 😭
```
<MazeMap
            campuses={111}
            zoom={14.5}
            height={hoverOnMap ? "450px" : "300px"}
            width={hoverOnMap ? "700px" : "500px"}
            center={{ lng: 151.23140898946815, lat: -33.91702431505671 }}
            onMapClick={(coords, zLevel) => {
              setMarkerCoordinates({
                lng: coords[0],
                lat: coords[1],
                zLevel: zLevel,
              });
            }}
            marker={{
              type: Marker.POIMarker,
              colour: '#ff0000',
              innerColour: '#0000ff',
              size: 32
            }}
            // having issues with marker and line, consult lachlan
          />
```
![image](https://github.com/user-attachments/assets/83985216-e0e6-406b-ac3a-c683b685174e)

**Comment by lachlanshoesmith** [comment]
that 'error' should only occur when u click on smth other than a room,, i can add a try catch to get rid of it but it should be harmless

**Comment by xleonx0x** [comment]
i dont see any marker tho 🤔

**Comment by lachlanshoesmith** [comment]
i see, likely cos it can't find a relevant poi
ill fix tonight when im home

**Comment by lachlanshoesmith** [comment]
fixed in c08e888
